app/pb_kits/playbook/pb_radio/radio.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbRadio
+    class Radio
+      include Playbook::Props
+
+      prop :dark, type: Playbook::Props::Boolean, default: false
+      prop :tag, type: Playbook::Props::String, default: "label"
+      prop :text, type: Playbook::Props::String, default: "Radio Item"
+      prop :name, type: Playbook::Props::String, default: "RadioGroup"
+      prop :value, type: Playbook::Props::String, default: "placeholder-value"
+      prop :checked
+      def classname
+        [
+          [
+            "pb_radio_kit",
+            dark_class,
+          ].compact.join("_"),
+          prop(:classname),
+        ].join(" ")
```

Comment:
There is a `generate_classname` method that will help you here. It does exactly what your doing here but you just pass it your unique classname parts. So this will become

```ruby
def classname
  generate_classname("pb_radio_kit", dark_class)
end
```